src/library/scala/collection/concurrent/CNodeBase.java

```diff
@@ -19,15 +19,15 @@
     public volatile int csize = -1;
 
     public boolean CAS_SIZE(int oldval, int nval) {
-	return updater.compareAndSet(this, oldval, nval);
+        return updater.compareAndSet(this, oldval, nval);
```

Comment:
Was this a TAB replaced with SPACES, or the other way around.

(we should have SPACES)